File: Logica de programação/Exercicios logica em C/03 - Estrutura repetitiva/Crescente/Crescente.c
#include <stdio.h>

int main(){

    int i, x, y;

    printf("Digite dois numeros: \n");
    fflush(stdin);
    scanf("%d", &x);
    scanf("%d", &y);

    while (x != y){
        if (x < y){
            printf("Crescente\n");
        } 
        else {
            printf("Decrescente\n");
        }

        printf("Digite outros dois numeros:\n");
        scanf("%d", &x);
        scanf("%d", &y);
    }

return 0;
}

File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_Java/Crescente/src/Crescente.java
import java.util.Scanner;
import java.util.Locale;

public class Crescente {
    public static void main(String[] args) throws Exception {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.println("Digite dois numeros: ");
        int x = sc.nextInt();
        int y = sc.nextInt();

        while (x != y) {

            if (x > y) {
                System.out.println("Decrescente");
            } else {
                System.out.println("Crescente");
            }

            System.out.println("Digite outros dois numeros: ");
            x = sc.nextInt();
            y = sc.nextInt();
        }

        sc.close();
    }
}
